src/scratch_continuous_category.js

```diff
@@ -0,0 +1,23 @@
+import * as Blockly from 'blockly/core';
+import {ContinuousCategory} from '@blockly/continuous-toolbox';
+
+class ScratchContinuousCategory extends ContinuousCategory {
+  createIconDom_() {
+    const icon = super.createIconDom_();
+    icon.style.border = `1px solid ${this.toolboxItemDef_['secondaryColour']}`;
+    return icon;
+  }
+  
+  setSelected(isSelected) {
+    super.setSelected(isSelected);
+    // Prevent hardcoding the background color to grey.
+    this.rowDiv_.style.backgroundColor = '';
```

Comment:
ugh are we actually doing that? why is all of our styling so baaaad. I'll make a note to file a bug in core. You shouldn't need to create a whole category class just to fix a few dumb CSS things.

Reply:
Yep: https://github.com/google/blockly-samples/blob/master/plugins/continuous-toolbox/src/ContinuousCategory.js#L48